Packs/Base/Scripts/CommonServerPython/CommonServerPython.py

```diff
@@ -2735,6 +2750,99 @@ def __init__(self, indicators=None, category=None, message=None):
         self.indicators_timeline = timelines
 
 
+
+def arg_to_int(arg, arg_name, required = False):
+    # type: (Any, str, bool) -> Optional[int]
+
+    """Converts an XSOAR argument to a Python int
+
+    This function is used to quickly validate an argument provided to XSOAR
+    via ``demisto.args()`` into an ``int`` type. It will throw a ValueError
+    if the input is invalid. If the input is None, it will throw a ValueError
+    if required is ``True``, or ``None`` if required is ``False.
+
+    :type arg: ``Any``
+    :param arg: argument to convert
+
+    :type arg_name: ``str``
```

Comment:
Why do we need `arg_name`? Just for printing the errors? It looks to me like something we can remove.

Reply:
Yes, to print errors. The purpose of this code was to provide easy way for developers to assert and convert their args/params. 
Without returning arg name or param name, we lose the purpose.
But instead I can just print `aa is not a valid number` `2012-22-34 is not a valid date/time range` 
What do you think @glicht ?

Reply:
Sounds good. Let's remove the arg_name or give it an empty value by default so this function can be used as a generic function.